docs/src/vertical_grids.md

```diff
@@ -121,7 +119,8 @@ But with the larger ``h / L`` is, the smaller the rate the spacings increase wit
 A ridiculously large value of ``h / L`` (approaching infinity) gives a uniform grid:
 
 ```@example vgrids
-z_faces(exponential_vertical_faces(; Nz, depth, scale = 1e20*depth))
+zgrid = exponential_vertical_faces(; Nz, depth, scale = 1e20*depth)
+z_faces(zgrid)
```

Comment:
what does "zgrid" and z_faces mean here?

Reply:
zgrid returns the ExponentialFaces type and z_faces builds the array of the faces given the type

I don't like it; but I'm trying to follow your suggestion to have a special type for each vertical grid. But perhaps in the course of doing that I'm mixing a bit of my thinking with your thinking....?